functions.php

```diff
@@ -1000,4 +1000,24 @@ function display_related_story( $story ) {
 	return ob_get_clean ();
 }
 
+/**
+* Replaces RSS description element content with a post's promo field if available.
+* If promo field is empty, the content is truncated to 30 words
+*
+* @author Cadie Brown
+* @return string
+*/
+function update_rss_description_to_promo( $content ) {
+	global $post;
+	$promoValue = get_post_meta($post->ID, 'promo', true);
```

Comment:
Nitpicky style guide stuff--underscores are preferred instead of camelCase for PHP variables (e.g. `$promo_value`